tasks.py

```diff
@@ -391,26 +367,30 @@ def pylint(context):
 @task(aliases=("a",))
 def autoformat(context):
     """Run code autoformatting."""
-    black(context, autoformat=True)
-    ruff(context, action="both", fix=True)
+    ruff(context, fix=True)
```

Comment:
The cookie uses `ruff(context, action=["format"], fix=True)`. Do we need to match that here?

Reply:
Good catch, I've been integrating changes from the cookie so yes!